src/components/SRNEntry.vue

```diff
@@ -55,7 +59,16 @@ export default {
     return {
       userIDList: [{ userID: "" }],
       invalidInputMessage: null,
+      validated: true,
       maxLength: 10,
+      /*this count is used to check how many times the user has been validated. 
```

Comment:
Don't use counts. Use an enum with string constants. Will make the code much more readable

Reply:
Using an enum `loginState : {
        UNAUTHENTICATED : 0,
        AUTHENTICATED : 1,
        INVALID : 2
      },`
      
  but I'll still need to maintain a global variable to compare with the enum, right?